README.md

```diff
@@ -80,7 +80,8 @@ To build SONiC installer image and docker images, run the following commands:
  **NOTE**:
 
 - Recommend reserving 50G free space to build one platform.
-- It is possible to run into errors where Docker's `/var/lib/docker` workspace folder can sit on a drive without enough empty space to do work (for example, untaring a large tar file). [Here is a link](https://linuxconfig.org/how-to-move-docker-s-default-var-lib-docker-to-another-directory-on-ubuntu-debian-linux) with information for moving the `var/lib/docker` folder to a drive with more disk space, as a workaround solution.
+- It is possible to run into errors where Docker's `/var/lib/docker` workspace folder can sit on a drive without enough empty space to do work (for example, untarring a large tar file). [Here is a link](https://linuxconfig.org/how-to-move-docker-s-default-var-lib-docker-to-another-directory-on-ubuntu-debian-linux) with information for moving the `var/lib/docker` folder to a drive with more disk space, as a workaround solution. One of these error messages can look something like:
+    - `/usr/bin/tar: ./datadrive/sonic-build-image/<some_file>: Cannot write: No space left on device`.
```

Comment:
"sonic-build-image" should be "sonic-buildimage". Also it should be noted that this error will occur while building a Docker image.

Also, can you reword to reduce the verbosity a bit?

Reply:
Sure, how's this look instead?

Reply:
It's better, but maybe something more like this:

- If Docker's workspace folder, `/var/lib/docker`, resides on a partition without sufficient free space, you may encounter an error like the following during a Docker container build job:

    `/usr/bin/tar: /path/to/sonic-buildimage/<some_file>: Cannot write: No space left on device`

    The solution is to [move the directory](https://linuxconfig.org/how-to-move-docker-s-default-var-lib-docker-to-another-directory-on-ubuntu-debian-linux) to a partition with more free space.

Reply:
Ah yeah I didn't realize you could just indent after the code excerpt. Nice

Reply:
Yeah, Markdown isn't all that intuitive.